### PR TITLE
Add a working example part

### PR DESCRIPTION
**Pull request by nicolas-bastien**
Hi, I would like to thank you for your bundle, I used it and a open source bundle to make a dashboard for sonata admin.

I saw there is not many forks and contributors to your project, maybe if we show working example it will give inspiration to other people, maybe some will add there link here too.

### CODE REVIEW

File: doc/usage.md [hunk 1]
```diff
@@ -106,3 +106,10 @@ $totalForAllResults = $response->getTotalsForAllResults();
 $hasRows = $response->hasRows();
 $rows = $response->getRows();
 ```
+
+# Working example
+
+If you want to have a working example of this bundle with symfony2 and sonata admin you can have a look at 
+[PrestaGoogleAnalyticsDashboardBundle](https://github.com/prestaconcept/PrestaGoogleAnalyticsDashboardBundle)
+
+
```
Comment (egeloen) [inline doc/usage.md]: Can you remove the extra blank line?

**Comment by egeloen** [comment]
:+1:

**Comment by nicolas-bastien** [comment]
Done, I will add a link to your bundle on my readme when it's ready.

And by the way I'm building a CMS based on Symfony CMF, if you want to have a look : https://github.com/prestaconcept/PrestaCMSCoreBundle

**Comment by egeloen** [comment]
@nicolas-bastien I have merged your change in 5b619d24e95c30d53b66317f5f3f3c03ad1b55d5 Thx!

**Pull request closed without merging**